File: examples/notes/update/node/updateANote.js
const { NotesService } = require("m3o/notes");

// Update a note
async function updateAnote() {
  let notesService = new NotesService(process.env.MICRO_API_TOKEN);
  let rsp = await notesService.update({
    note: {
      id: "63c0cdf8-2121-11ec-a881-0242e36f037a",
      text: "Updated note text",
      title: "Update Note",
    },
  });
  console.log(rsp);
}

updateAnote();

File: examples/go/notes/update/updateANote.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/notes"
)

// Update a note
func UpdateAnote() {
	notesService := notes.NewNotesService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := notesService.Update(&notes.UpdateRequest{
		Note: &notes.Note{
			Id:    "63c0cdf8-2121-11ec-a881-0242e36f037a",
			Text:  "Updated note text",
			Title: "Update Note",
		},
	})
	fmt.Println(rsp, err)
}
